MetaMask/metamask-extension
# UX: Show Checksum Addresses in Account Menu

## Explanation

@kumavis pointed out that the account menu wasn't using a checksum'd address, so the casing was different than the copy button on the home screen.  This PR fixes that issue.

## Screenshots/Screencaps

<img width="424" alt="SCR-20230721-qoou" src="https://github.com/MetaMask/metamask-extension/assets/46655/4b8b045e-f2d8-48be-b739-17c6233a2b8e">


## Manual Testing Steps

1.  On the home screen, in the CopyAddressButton, see some capital letters
2. Click the Accounts menu
3. See the same letters capitalized

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
